### PR TITLE
Add five FOCI'24 papers and correct the year of a publication

### PR DESCRIPTION
**Pull request by gfw-report**
This pull request adds three papers and two extended abstracts from FOCI'24. It also corrects the year of a publication.

* Homepage: https://foci.community/foci24.html
* Publications: https://www.petsymposium.org/foci/2024/
* Discussion group: https://github.com/net4people/bbs/issues/329

### CODE REVIEW

File: references.bib [hunk 1]
```diff
@@ -1,4 +1,58 @@
-@inproceedings{Xue2023b,
+@inproceedings{2024a,
```
Comment (gaukas) [inline references.bib:1]:
```suggestion
@inproceedings{Tsai2024a,
```

Maybe a typo?

File: references.bib [hunk 2]
```diff
@@ -1,4 +1,58 @@
...
+	title = {Modeling and Detecting {Internet} Censorship Events},
+	booktitle = {Network and Distributed System Security},
+	publisher = {The Internet Society},
+	year = {2024},
+	url = {https://www.ndss-symposium.org/wp-content/uploads/2024-409-paper.pdf},
+}
+
+@inproceedings{Kristoff2024a,
+	author = {John Kristoff and Max Resing and Moritz M\"{u}eller and Arturo Filast\`{o} and Chris Kanich and Niels ten Oever},
+	title = {{Internet} Sanctions on {Russian} Media: Implementation and Effects},
```
Comment (NullHypothesis) [inline references.bib:12]: That's a typo, no?
```suggestion
	title = {{Internet} Sanctions on {Russian} Media: Actions and Effects},
```

File: references.bib [hunk 3]
```diff
@@ -1,4 +1,58 @@
...
+	author = {Elisa Tsai and Ram Sundara Raman and Atul Prakash and Roya Ensafi},
+	title = {Modeling and Detecting {Internet} Censorship Events},
+	booktitle = {Network and Distributed System Security},
+	publisher = {The Internet Society},
+	year = {2024},
+	url = {https://www.ndss-symposium.org/wp-content/uploads/2024-409-paper.pdf},
+}
+
+@inproceedings{Kristoff2024a,
+	author = {John Kristoff and Max Resing and Moritz M\"{u}eller and Arturo Filast\`{o} and Chris Kanich and Niels ten Oever},
```
Comment (NullHypothesis) [inline references.bib:11]: AIUI, the author order is based on rows, not columns.
```suggestion
	author = {John Kristoff and Moritz M\"{u}eller and Arturo Filast\`{o} and Max Resing and Chris Kanich and Niels ten Oever},
```

File: references.bib [hunk 4]
```diff
@@ -1,4 +1,58 @@
...
+	title = {Bleeding Wall: A Hematologic Examination on the {Great Firewall}},
+	booktitle = {Free and Open Communications on the Internet},
+	publisher = {},
+	year = {2024},
+	url = {https://www.petsymposium.org/foci/2024/foci-2024-0002.pdf},
+}
+
+@inproceedings{Chi2024a,
+	author = {Erik Chi and Gaukas Wang and J. Alex Halderman and Eric Wustrow and Jack Wampler},
+	title = {Just add {WATER}: WebAssembly-based Circumvention Transports},
```
Comment (NullHypothesis) [inline references.bib:30]:
```suggestion
	title = {Just add {WATER}: {WebAssembly}-based Circumvention Transports},
```

File: references.bib [hunk 5]
```diff
@@ -1,4 +1,58 @@
...
+	url = {https://www.petsymposium.org/foci/2024/foci-2024-0004.pdf},
+}
+
+@inproceedings{Chen2024a,
+	author = {Mingye Chen and Jack Wampler and Abdulrahman Alaraj and Gaukas Wang and Eric Wustrow},
+	title = {Extended Abstract: {Oscur0}: One-shot Circumvention without registration},
+	booktitle = {Free and Open Communications on the Internet},
+	publisher = {},
+	year = {2024},
+	url = {https://www.petsymposium.org/foci/2024/foci-2024-0005.php},
```
Comment (NullHypothesis) [inline references.bib:52]:
```suggestion
	url = {https://www.petsymposium.org/foci/2024/foci-2024-0005.pdf},
```

File: references.bib [hunk 6]
```diff
@@ -1,4 +1,58 @@
...
+	title = {Extended Abstract: Traffic Splitting for Pluggable Transports},
+	booktitle = {Free and Open Communications on the Internet},
+	publisher = {},
+	year = {2024},
+	url = {https://www.petsymposium.org/foci/2024/foci-2024-0004.pdf},
+}
+
+@inproceedings{Chen2024a,
+	author = {Mingye Chen and Jack Wampler and Abdulrahman Alaraj and Gaukas Wang and Eric Wustrow},
+	title = {Extended Abstract: {Oscur0}: One-shot Circumvention without registration},
```
Comment (NullHypothesis) [inline references.bib:48]:
```suggestion
	title = {Extended Abstract: {Oscur0}: One-shot Circumvention without Registration},
```

**Review by NullHypothesis — CHANGES_REQUESTED** [review]
Left a few suggestions. As always, thanks a lot for your help! :heart:

**Comment by gfw-report** [comment]
Thank you so much for your careful review, @NullHypothesis ! I have fixed them all now. Most of the changes were needed because I based it on an unfinalized version of the publication list: https://github.com/net4people/bbs/issues/329. I will make sure to use the camera-ready version in the future.

File: header.tpl [hunk 7]
```diff
@@ -188,7 +188,7 @@
           </div>
           <div class="menu-item">
             <img class="top-icon" src="img/update-icon.svg" alt="update icon"/>
-            <a href="https://github.com/NullHypothesis/censorbib/commits/master">Last update: 2024-02-17</a>
+            <a href="https://github.com/NullHypothesis/censorbib/commits/master">Last update: 2024-02-25</a>
```
Comment (NullHypothesis) [inline header.tpl:191]: You leaked a few bits of your geolocation because it's still 2024-02-24 where I live 😉

**Review by NullHypothesis — APPROVED** [review]
Looks good, thanks again!

**Pull request merged**